src/cljam/vcf/reader.clj

```diff
@@ -93,28 +94,32 @@
 ;; ------------------
 
 (defn- parse-data-line
-  [line header]
-  (let [[fields gt-fields] (->> (cstr/split line #"\t")
+  [line header kws]
+  (let [[fields gt-fields] (->> (.split ^String line "\t")
+                                clojure.lang.LazilyPersistentVector/createOwning
```

Comment:
These two lines have potentially degradation risk. Please add comment describing that this implementation is for performance issue.